### PR TITLE
[DOCS] How to validate data by running a Checkpoint

### PR DESCRIPTION
**Pull request by alexsherstinsky**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
- JIRA: GREAT-200/GREAT-225
-
-

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added [unit tests](https://docs.greatexpectations.io/en/latest/contributing/testing.html#contributing-testing-writing-unit-tests) where applicable and made sure that new and existing tests are passing.
- [x] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: docs/guides/validation/how_to_validate_data_by_running_a_checkpoint.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+- Configured a [Data Context](../../tutorials/getting_started/initialize_a_data_context.md).
+- Configured an [Expectations Suite](../../tutorials/getting_started/create_your_first_expectations.md).
+- Configured a [Checkpoint](./checkpoints/how_to_create_a_new_checkpoint)
+
+</Prerequisites>
+
+Steps
+-----
+
+1. You can update your existing checkpoint either using a text editor or by executing the following command:
```
Comment (cdkini) [inline docs/guides/validation/how_to_validate_data_by_running_a_checkpoint.md]: Could we please capitalize checkpoint just for consistency?

**Review by cdkini — APPROVED** [review]
Very minor capitalization comment but LGTM!

Thanks for writing this up ✏️

File: docs/guides/validation/how_to_validate_data_by_running_a_checkpoint.md (hunk 1 above)
Comment (alexsherstinsky) [reply docs/guides/validation/how_to_validate_data_by_running_a_checkpoint.md]: @cdkini Nice catch -- fixed.  Thank you!

**Review by Shinnnyshinshin — APPROVED** [review]
Looks good to me as well. I also wanted to ask : what is the plan for getting Checkpoints + checkpoint configurations tested as part of the Docs Integration tests? I understand that this Doc uses the CLI and refers to other docs for additional details, but I was wondering where/if those tests were intended to live

**Comment by abegong** [comment]
Here's a review by video: https://www.loom.com/share/4eb45a9550cf49f8ba415e6fc99c68f6

Next steps:
	1. Fix the prerequisites section. @donaldheppner , @spencerhardwick , let's sync up on this.
	2. Drop or move the content in steps 1 and 2
	3. Pull the articles in step 3 into the doc itself, put them in tabs, and delete the originals
	4. Add a Core Promises section, as described in the video

**Review by abegong — CHANGES_REQUESTED** [review]
See previous comment...

**Review by abegong — APPROVED** [review]
Thanks for making these changes, @alexsherstinsky . This is much improved.

There are still several things to do to improve the doc, but we can handle them in a subsequent PR.

**Pull request merged**